namespace/proposal-04-extend.adoc

```diff
@@ -0,0 +1,34 @@
+# Proposal 3: Extend javax in Jakarta EE 9 and beyond
```

Comment:
Proposal 3 or 4?  :-)  We need to resolve whether Richard's proposal is really different enough from the existing proposals.

Reply:
Yep, already found that one with my second commit...  :-)